test/e2e/network/ingress.go

```diff
@@ -947,50 +947,53 @@ var _ = SIGDescribe("Ingress API", func() {
 		Testname: Ingress API
 		Description:
 		The networking.k8s.io API group MUST exist in the /apis discovery document.
-		The networking.k8s.io/v1beta1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
-		The ingresses resources MUST exist in the /apis/networking.k8s.io/v1beta1 discovery document.
+		The networking.k8s.io/v1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
+		The ingresses resources MUST exist in the /apis/networking.k8s.io/v1 discovery document.
 		The ingresses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
 		The ingresses/status resource must support update and patch
-
 	*/
 
-	ginkgo.It("should support creating Ingress API operations", func() {
+	framework.ConformanceIt("should support creating Ingress API operations", func() {
 		// Setup
 		ns := f.Namespace.Name
-		ingVersion := "v1beta1"
-		ingClient := f.ClientSet.NetworkingV1beta1().Ingresses(ns)
-
-		prefixPathType := networkingv1beta1.PathTypePrefix
+		ingVersion := "v1"
+		ingClient := f.ClientSet.NetworkingV1().Ingresses(ns)
+
+		prefixPathType := networkingv1.PathTypePrefix
+		serviceBackend := &networkingv1.IngressServiceBackend{
+			Name: "defaultbackend",
+			Port: networkingv1.ServiceBackendPort{
+				Name:   "",
+				Number: 80,
+			},
+		}
+		defaultBackend := networkingv1.IngressBackend{
+			Service: serviceBackend,
+		}
 
-		ingTemplate := &networkingv1beta1.Ingress{
+		ingTemplate := &networkingv1.Ingress{
 			ObjectMeta: metav1.ObjectMeta{GenerateName: "e2e-example-ing",
 				Labels: map[string]string{
 					"special-label": f.UniqueName,
 				}},
-			Spec: networkingv1beta1.IngressSpec{
-				Backend: &networkingv1beta1.IngressBackend{
-					ServiceName: "default-backend",
-					ServicePort: intstr.FromInt(8080),
-				},
-				Rules: []networkingv1beta1.IngressRule{
+			Spec: networkingv1.IngressSpec{
+				DefaultBackend: &defaultBackend,
+				Rules: []networkingv1.IngressRule{
 					{
 						Host: "foo.bar.com",
-						IngressRuleValue: networkingv1beta1.IngressRuleValue{
-							HTTP: &networkingv1beta1.HTTPIngressRuleValue{
-								Paths: []networkingv1beta1.HTTPIngressPath{{
+						IngressRuleValue: networkingv1.IngressRuleValue{
+							HTTP: &networkingv1.HTTPIngressRuleValue{
+								Paths: []networkingv1.HTTPIngressPath{{
 									Path:     "/",
 									PathType: &prefixPathType,
-									Backend: networkingv1beta1.IngressBackend{
-										ServiceName: "test-backend",
-										ServicePort: intstr.FromInt(8080),
-									},
+									Backend:  defaultBackend,
```

Comment:
keep this as equivalent to the existing test as possible, e.g. use `test-backend`, not defaultBackend